jenkins/osa_onmetal.functions.groovy

```diff
@@ -352,8 +352,8 @@ def start_api_uptime_tests() {
     ssh -o StrictHostKeyChecking=no root@${host_ip} '''
     sudo rm -f /usr/api.uptime.stop
     cd api_uptime/api_uptime
-    python call_test.py -v
-    '''
+    python call_test.py -d -s nova -o ../../output
```

Comment:
Checked with @lamarwhitej and the "during" test is doing nova, swift, keystone so I will update to match.

For the output path the "during" tests the file goes into the same directory as the project.  Since my github repo is structured differently than his, mine goes up 2 directories when saving the output so that its in the same project root as the "during" tests.  Makes it easier to gather the output files during post-processing.